templates/options.conf.erb

```diff
@@ -4,3 +4,6 @@ forwarders { <%= scope.lookupvar('::dns::forwarders').join("; ") %>; };
 <% end -%>
 
 listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
+<% if (@operatingsystem == 'FreeBSD') -%>
```

Comment:
But this is the pid file. I'd expect that it would be compiled with the sane default.